fuse/dl/models/model_multihead.py

```diff
@@ -54,6 +54,12 @@ def __init__(
             raise Exception(
                 "Neither conv_inputs nor backbone_args are None. One must be set (conv_inputs soon to be deprecated)"
             )
+        # we don't use the number of input channels.
+        # it's just kept for backward compatibility
+        if isinstance(conv_inputs[0], Tuple):
```

Comment:
conv_inputs (and also backbone_args) might be None (and maybe also empty?).
What do you think about:
```
if conv_inputs is not None:
    self.conv_inputs = [conv_input[0] for conv_input in self.conv_inputs if isinstance(conv_input, Tuple) else conv_input]
```

Reply:
sounds good. just the syntax should be:
`[inp[0]  if isinstance(inp, Tuple) else inp for inp in self.conv_inputs]`